pages/common/ansible-doc.md

```diff
@@ -0,0 +1,29 @@
+# ansible-doc
+
+> Display information on modules installed in Ansible libraries.
+> It displays a terse listing of plugins and their short descriptions.
+> More information: <https://docs.ansible.com/ansible/latest/cli/ansible-doc.html>.
+
+- List available action plugins (modules):
+
+`ansible-doc -l`
```

Comment:
Is there a long-form of this argument? We generally prefer long-forum arguments to aid clarity in tldr pages.

Same goes for the other examples.